src/graph/algo.rs

```diff
@@ -247,23 +260,36 @@ pub fn scroll_offset<I: GraphIndex>(graph: &Graph, idx: I) -> Point {
                     let is_already_offset = |mut position_parents: super::RecursiveWalk<_>| {
                         while let Some(position_parent) = position_parents.next_node(graph) {
                             if graph.depth_parent_recursion(position_parent)
-                                .any(graph, |_g, _e, n| n == depth_parent)
-                            {
+                                    .any(graph, |_g, _e, n| n == depth_parent) {
                                 return true;
                             }
                         }
                         false
                     };
 
-                    let x_offset = depth_parent_widget.maybe_x_scroll_state.map(|scroll| {
-                        let position_parents = graph.x_position_parent_recursion(idx);
-                        if is_already_offset(position_parents) { 0.0 } else { scroll.offset }
-                    }).unwrap_or(0.0);
-
-                    let y_offset = depth_parent_widget.maybe_y_scroll_state.map(|scroll| {
-                        let position_parents = graph.y_position_parent_recursion(idx);
-                        if is_already_offset(position_parents) { 0.0 } else { scroll.offset }
-                    }).unwrap_or(0.0);
+                    let x_offset = depth_parent_widget.maybe_x_scroll_state
+                                                      .map(|scroll| {
```

Comment:
I think these "align the next method with the next dot" changes are generally pretty poor and waste a lot of whitespace, especially for methods that take closures with big bodies like this. This particular section seems to become quite messy and turns from 9 concise lines to 23. I wonder if there's some kind of rustfmt config for fixing this?